src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,25 @@
+package calculator;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class StringCalculatorTest {
+
+    @Test
+    @DisplayName("주어진 elements대로 계산해야 한다")
```

Comment:
`elements` 와 같은 구체적인 변수명은 언제든 변할 수 있는 것이라고 생각해요.
그렇기 때문에 일반적으로 사전적의미를 갖는 명칭을 사용하는건 어떨까요?

```suggestion
    @DisplayName("연산자에 맞게 계산한다")
```